cosmos/constants.py

```diff
@@ -75,6 +75,7 @@ class TestBehavior(Enum):
     Behavior of the tests.
     """
 
+    BUILD = "build"
```

Comment:
shall we update docs: https://github.com/astronomer/astronomer-cosmos/blob/main/docs/configuration/testing-behavior.rst

Reply:
Addressed in 20ba9c7ae9fdb68e0d2c0c87f1ab662b5ba96402, with minor fixes in commits afterwards.

You can see the docs rendered in via Cloudflare:
https://585d1321.astronomer-cosmos.pages.dev/configuration/testing-behavior/